lib/github-package.js

```diff
@@ -415,9 +419,12 @@ export default class GithubPackage {
   @autobind
   async cloneRepositoryForProjectPath(remoteUrl, projectPath) {
     const context = this.contextPool.getContext(projectPath);
+    // TODO: do we need to pass pipelineManager? or is this discarded immediately & recreated later in workdir context
```

Comment:
The `Repository` created here is indeed just discarded immediately; the "real" one will be created later by `getNextContext()`, either triggered by the `project.addPath()` call down below, or when you open a file in the new project.

But, I'd still give it a pipelineManager just the same. It makes it easier to reason about because we never have to worry about whether or not a given `Repository` has a manager or not.